build.gradle

```diff
@@ -277,14 +277,16 @@ task docs(type: Javadoc) {
 def setJavaDocOptions(MinimalJavadocOptions options) {
     options.author = true
     options.version = true
-    options.links 'http://docs.oracle.com/javase/7/docs/api/'
+    options.links 'https://docs.oracle.com/javase/9/docs/api/'
```

Comment:
Now, we have link to java package : 

![image](https://user-images.githubusercontent.com/1149149/39409865-6fd83a04-4bee-11e8-80f1-21799a640324.png)

 Not working without https : 

![image](https://user-images.githubusercontent.com/1149149/39409873-8c41e3d4-4bee-11e8-900e-922d28296ff0.png)